docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,68 +1,189 @@
+The codebase SHOULD document the governance of the codebase, contributions and its community, for example in a GOVERNANCE file. | no |
+The codebase MAY include a code of conduct for contributors. |  |
 
-## [Welcome contributors](https://standard.publiccode.net/criteria/open-to-contributions.html)
 
-* Improve [CONTRIBUTING.md](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md), perhaps with content from Amsterdam's [CONTRIBUTING.md](https://github.com/Amsterdam/amsterdam.github.io/blob/master/CONTRIBUTING.md)
 
 ## [Maintain version control](https://standard.publiccode.net/criteria/version-control-and-history.html)
 
-* Good.
-    * Some past commits did not state what they do or why, some just reference an issue, or have "[WIP](https://github.com/Amsterdam/signals/commit/2f9e2f73ee5dc4cdf67d6854d1a7361f6e6aaf9b)" commit messages; developers have become more rigorous.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+You MUST have a way to maintain version control for your code. | yes |
+All files in a codebase MUST be version controlled. | yes |
+All decisions MUST be documented in commit messages. | no | Commit messages often refer to private issues.
+Every commit message MUST link to discussions and issues wherever possible. | partial | Sometimes these are private connections.
+You SHOULD group relevant changes in commits. | yes | Commits are grouped by pull request.
+You SHOULD mark different versions of your codebase, for example using revision tags or textual labels. | yes | Amsterdam tags new releases on Github using Github releases.
+You SHOULD prefer file formats that can easily be version controlled. | yes | Git-friendliness is in the culture.
 
 ## [Require review of contributions](https://standard.publiccode.net/criteria/require-review.html)
 
-* Ensure commits are reviewed including commit message.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All contributions that are accepted or committed to release versions of the codebase MUST be reviewed by another contributor. | yes |
+Reviews MUST include source, policy, tests and documentation. | | Code yes, but tests and docs/policy not always - maybe a PR template would help?
+Reviewers MUST provide feedback on all decisions made and the implementation in the review. | yes | Weekly dev call for discussion.
+Contributions SHOULD conform to the standards, architecture and decisions set out in the codebase in order to pass review. | yes | Architecture docs for both frontend and backend, might be nice to have a checkbox in a PR template.
+Reviews SHOULD include running both the code and the tests of the codebase. | yes |
+Contributions SHOULD be reviewed by someone in a different context than the contributor. | some | VNG pull requests all reviewed by Amsterdam, frontend & backend cross-review rare, Amsterdam pull requests often not cross-reviewed by VNG. PO acceptance testing not yet part of review.
```

Comment:
Regarding PO acceptance testing, this is part of the flow in Amsterdam (where relevant, not every change is user, and thus PO, visible). Our SCRUM board has a review and an acceptance testing lane --- both are hit before deployment to production.